xml/obs_scm_ci_workflow_integration.xml

```diff
@@ -0,0 +1,1044 @@
+
+          <para><emphasis role="bold">b) Revoke the token</emphasis></para>
+
+          <para>You can always revoke your token via WebUI, in
+          <emphasis>Profile &gt; Manage Your Tokens</emphasis>, or with these
+          commands:</para>
+
+          <screen language="xml"><command>osc</command> token # list all your tokens</screen>
+
+          <screen language="xml"><command>osc</command> token --delete $token_id # remove the token with the given id</screen>
+
+          <para>Then you can create a new one as explained in <xref
+          linkend="sec.obs.obs_scm_ci_workflow_integration.setup.token_authentication.how_to_authenticate_scm_with_obs"/>
+          and replace it wherever you use it.</para>
+        </sect4>
+      </sect3>
+    </sect2>
+
+    <sect2 xml:id="sec.obs.obs_scm_ci_workflow_integration.setup.webhooks">
+      <title>Webhooks</title>
+
+      <para>Once OBS and your SCM are allowed to speak to each other, they can
+      start talking via <emphasis role="bold">webhooks</emphasis>.</para>
+
+      <sect3 xml:id="sec.obs.obs_scm_ci_workflow_integration.setup.webhooks.how_to_set_up_a_webhook_on_github">
+        <title>How to Set Up a Webhook on Github</title>
+
+        <para>Go to the project you want to set the integration on, then under
+        <emphasis>Settings &gt; Webhooks</emphasis>.</para>
+
+        <figure>
+          <title>Webhooks on GitHub.</title>
+
+          <mediaobject>
+            <imageobject>
+              <imagedata fileref="https://openbuildservice.org/images/posts/sprint_95_github_webhook_form.png"/>
+            </imageobject>
+          </mediaobject>
+        </figure>
+
+        <para>You have to fill in the form with:</para>
+
+        <itemizedlist>
+          <listitem>
+            <para><emphasis role="bold">Payload URL:</emphasis>
+            <emphasis>https://build.opensuse.org/trigger/workflow?id=12345</emphasis>.
+            Replace 12345 with the OBS token numerical ID previsously
+            obtained.</para>
+          </listitem>
+
+          <listitem>
+            <para><emphasis role="bold">Content type:</emphasis>
+            application/json.</para>
+          </listitem>
+
+          <listitem>
+            <para><emphasis role="bold">Secret:</emphasis>
+            <emphasis>uvwxyz</emphasis>. Replace uvwxyz with the OBS token
+            secret string previously obtained.</para>
+          </listitem>
+
+          <listitem>
+            <para><emphasis role="bold">Enable SSL
+            verification</emphasis>.</para>
+          </listitem>
+
+          <listitem>
+            <para><emphasis role="bold">Let me select individual
+            events</emphasis>: Pull requests, Pushes.</para>
+          </listitem>
+        </itemizedlist>
+      </sect3>
+
+      <sect3 xml:id="sec.obs.obs_scm_ci_workflow_integration.setup.webhooks.how_to_set_up_a_webhook_on_gitlab">
+        <title>How to Set Up a Webhook on GitLab</title>
```

Comment:
Are you going to concentrate on github or not? :)

Reply:
Yes we want to use GitHub to avoid the duplication. We documented how to set up the webhook for GItHub in the previous section, but doing it for GitLab is no duplication, as it is a bit different.